scripts/globals/combat/physical_utilities.lua

```diff
@@ -47,6 +47,31 @@ local wsElementalProperties =
     [      16] = {    0,    0,   1,    0,     1,       0,     1,     0,    1 }, -- Lv 4, Darkness
 }
 
+-- Table with pDIF caps per weapon/skill type.
+local pDifWeaponCapTable =
```

Comment:
This table looks odd. 
"1H melee weapons have an effective ratio cap of 3.625" and yet the charts show different numbers.  I wonder if this "effective" cap is including Damage Limit bonus, and the limits should be changed to match the table `Pre-Randomizer pDIF Caps by Weapon Type`

Reply:
From what I could gather...
Column 1: the "effective cap" is pre-level diff and wratio
Column 2: after that, but before melee random factor and ranged distance correction.

The damage limit+ is a job trait. aka, a modifier. I dont remember the exact math, but it was dead simple...
newCap = oldCap + 0.2 * trait_level <-- something like that, from memory. Bottom line, no need to table it.